src/settings/JobProfiles/JobProfiles.js

```diff
@@ -26,6 +26,85 @@ import { JobProfilesForm } from './JobProfilesForm';
 const INITIAL_RESULT_COUNT = 5000;
 const RESULT_COUNT_INCREMENT = 5000;
 
+export const jobProfilesShape = {
+  INITIAL_RESULT_COUNT,
+  RESULT_COUNT_INCREMENT,
+  manifest: {
+    initializedFilterConfig: { initialValue: false },
+    query: { initialValue: {} },
+    resultCount: { initialValue: INITIAL_RESULT_COUNT },
+    records: {
+      type: 'okapi',
+      perRequest: RESULT_COUNT_INCREMENT,
+      records: ENTITY_KEYS.JOB_PROFILES,
+      recordsRequired: '%{resultCount}',
+      path: 'data-import-profiles/jobProfiles',
+      clientGeneratePk: false,
+      throwErrors: true,
+      GET: {
+        params: {
+          query: makeQueryFunction(
+            'cql.allRecords=1',
+            '(name="%{query.query}*" OR tags.tagList="%{query.query}*")',
+            {
+              name: 'name',
+              tags: 'tags.tagList',
+              updated: 'metadata.updatedDate',
+              updatedBy: 'userInfo.firstName userInfo.lastName userInfo.userName',
+            },
+            [],
+          ),
+        },
+        staticFallback: { params: {} },
+      },
+    },
+  },
+  visibleColumns: [
+    'name',
+    'description',
+    'tags',
+    'updated',
+    'updatedBy',
+  ],
+  columnWidths: {
+    isChecked: '35px',
+    name: '300px',
+    tags: '150px',
+    updated: '100px',
+    updatedBy: '250px',
+  },
+  renderHeaders: props => {
+    let headers = {
+      name: <FormattedMessage id="ui-data-import.name" />,
+      description: <FormattedMessage id="ui-data-import.description" />,
+      tags: <FormattedMessage id="ui-data-import.tags" />,
+      updated: <FormattedMessage id="ui-data-import.updated" />,
+      updatedBy: <FormattedMessage id="ui-data-import.updatedBy" />,
+    };
+
+    if (props && props.checkboxList) {
```

Comment:
I would use default assigning like `renderHeaders: (props = {}) => {`, so no additional checks for it is needed. The is also true for [here](https://github.com/folio-org/ui-data-import/pull/320/files#diff-f006c40166676f423d25786d486ab283R95), [here](https://github.com/folio-org/ui-data-import/pull/320/files#diff-e7ac3d689217081429bf909a5ddae36fR92) and [here](https://github.com/folio-org/ui-data-import/pull/320/files#diff-6da8b35c659aa25cd4ef88ca4e5f2cf4R90-R96).